pyvista/plotting/plotter.py

```diff
@@ -2207,8 +2207,8 @@ def key_press_event(self, *args, **kwargs) -> None:  # numpydoc ignore=PR01,RT01
     def left_button_down(self, *args):
         """Register the event for a left button down click."""
         if (
-            hasattr(self.render_window, 'GetOffScreenFramebuffer')
-            and not self.render_window.GetOffScreenFramebuffer().GetFBOIndex()  # type: ignore[union-attr]
+            hasattr(self.render_window, 'GetRenderFramebuffer')
+            and not self.render_window.GetRenderFramebuffer().GetFBOIndex()  # type: ignore[union-attr]
```

Comment:
Deprecated from vtk 9.1. See removal in 9.3 [here](https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9300/diffs#diff-content-6cf09b948f42b890a33c2e2222778843b8b87810) and [here](https://docs.vtk.org/en/latest/release_details/9.3.html#legacy)